src/ore/Cargo.toml

```diff
@@ -11,6 +11,7 @@ default = ["network", "chrono", "cli", "metrics", "stack", "test"]
 cli = ["clap"]
 metrics = ["prometheus"]
 network = ["async-trait", "bytes", "futures", "openssl", "smallvec", "tokio-openssl", "tokio"]
+tokio_spawn = ["tokio", "tokio/tracing"]
```

Comment:
These features are named after the modules they enable (with the exception of `network`; that should get adjusted), so let's call this `task`.